scripts/migration/migratenodestatus.py

```diff
@@ -0,0 +1,10 @@
+from JumpScale import j
+ccl = j.clients.osis.getNamespace('cloudbroker')
+ocl = j.clients.osis.getNamespace('system')
+nodes_ids = ocl.node.list()
+for node_id in nodes_ids:
+    node = ocl.node.get(node_id)
+    if not node.status:
+        stack = ccl.stack.get(node.id)
```

Comment:
stack = ccl.stack.searchOne({'referenceId': str(node.id)})
if stack ...

Reply:
this script shoudle be under https://github.com/0-complexity/openvcloud/tree/master/libs/agent-scripts/cloudbroker/upgrades/2.3.1/master

can even be part of the existint upgrademodel script